common/pom.xml

```diff
@@ -56,7 +56,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>114</maxAllowedViolations>
+          <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
I do see different behavior depending on whether logViolationsToConsole or consoleOutput is set to true in the pom file. Without consoleOutput set to true only total number of violations is printed.